src/catalog/table_catalog.cpp

```diff
@@ -561,6 +561,24 @@ TableCatalog::GetTableObjects(concurrency::TransactionContext *txn) {
   return database_object->GetTableObjects();
 }
 
+/*@brief   read table catalog objects from pg_table using database oid
```

Comment:
Comments are not consistent with the actual function. Searching using the schema name instead of db oid